lib/spack/spack/cmd/env.py

```diff
@@ -353,7 +353,8 @@ def env_status(args):
 # env loads
 #
 def env_loads_setup_parser(subparser):
-    """list modules for an installed environment '(see spack module loads)'"""
+    """list modules for an installed environment
+'(see spack module tcl loads)'"""
```

Comment:
This will cause a newline to be added to the help message. We usually try to avoid this. Maybe we don't need the cross-reference anymore? Or maybe we could add it as an epilog? I'll let other maintainers chime in.

Reply:
fwiw, I initially put this on the same line, but CI failed because the line was too long.

Reply:
Yeah, it likes to do that. We could add a backslash to prevent the newline, but then the question is whether the help message is too long.